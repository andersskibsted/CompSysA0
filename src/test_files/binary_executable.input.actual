test_files/binary_executable.input: data
