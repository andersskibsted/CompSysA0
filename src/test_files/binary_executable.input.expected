test_files/binary_executable.input: Mach-O 64-bit executable arm64
